frontend/src/pages/CollaborationReportForm/index.js

```diff
@@ -0,0 +1,549 @@
+/*
+  Collaboration report. Makes use of the navigator to split the long form into
+  multiple pages. Each "page" is defined in the `./Pages` directory.
+*/
+import React, {
+  useState, useEffect, useRef, useContext, useMemo,
+} from 'react';
+import PropTypes from 'prop-types';
+import { startCase, keyBy, mapValues } from 'lodash';
+import { Helmet } from 'react-helmet';
+import ReactRouterPropTypes from 'react-router-prop-types';
+import { useHistory, Redirect } from 'react-router-dom';
+import { Alert, Grid } from '@trussworks/react-uswds';
+import { REPORT_STATUSES, DECIMAL_BASE } from '@ttahub/common';
+import moment from 'moment';
+import useDeepCompareEffect from 'use-deep-compare-effect';
+import pages from './Pages';
+import ActivityReportNavigator from '../../components/Navigator/ActivityReportNavigator';
+import './index.scss';
+import { NOT_STARTED } from './constants';
+import {
+  LOCAL_STORAGE_CR_DATA_KEY,
+  LOCAL_STORAGE_CR_ADDITIONAL_DATA_KEY,
+  LOCAL_STORAGE_CR_EDITABLE_KEY,
+} from '../../Constants';
+import { getRegionWithReadWrite } from '../../permissions';
+import useTTAHUBLocalStorage from '../../hooks/useTTAHUBLocalStorage';
+// import {
+//   submitReport,
+//   saveReport,
+//   getReport,
+//   getRecipientsForExistingAR,
+//   createReport,
+//   getCollaborators,
+//   getApprovers,
+//   reviewReport,
+//   resetToDraft,
+//   getGroupsForActivityReport,
+//   getRecipients,
+// } from '../../fetchers/activityReports';
+import { getCollaborators } from '../../fetchers/collaboratorReports';
+import useLocalStorage, { setConnectionActiveWithError } from '../../hooks/useLocalStorage';
+import NetworkContext, { isOnlineMode } from '../../NetworkContext';
+import UserContext from '../../UserContext';
+import MeshPresenceManager from '../../components/MeshPresenceManager';
+
+const defaultValues = {
+  ECLKCResourcesUsed: [],
+  activityRecipientType: '',
+  activityRecipients: [],
+  activityType: [],
+  additionalNotes: null,
+  files: [],
+  collaborators: [],
+  activityReportCollaborators: [],
+  context: '',
+  deliveryMethod: null,
+  duration: '',
+  endDate: null,
+  goals: [],
+  recipientNextSteps: [{ id: null, note: '' }],
+  recipients: [],
+  nonECLKCResourcesUsed: [],
+  numberOfParticipants: null,
+  objectivesWithoutGoals: [],
+  otherResources: [],
+  participantCategory: '',
+  participants: [],
+  reason: [],
+  requester: '',
+  specialistNextSteps: [{ id: null, note: '' }],
+  startDate: null,
+  calculatedStatus: REPORT_STATUSES.DRAFT,
+  targetPopulations: [],
+  topics: [],
+  approvers: [],
+  recipientGroup: null,
+  language: [],
+};
```

Comment:
these look like they may need an update

Reply:
Yeah, that's straight copy-pasted from the AR form, forgot to remove/clean up. But this PR is mainly for the Activity Summary page, so my plan is to clean all that up later when I work on making the CRForm save and submit and such.